react/NestedSelect/NestedSelect.md

```diff
@@ -0,0 +1,110 @@
+```
+import Button from '../Button'
+import Circle from '../Circle'
+import NestedSelectModal from './Modal';
+import { useState } from 'react'
+
+
+const Image = ({ letter }) => (
+  <Circle backgroundColor='var(--melon)'>
+    { letter }
+  </Circle>
+)
+
+const options = {
+  children: [
+    {
+      title: 'A',
+      icon: <Image letter='A' />
+    },
+    {
+      title: 'B',
+      icon: <Image letter='B' />, 
+      children: [
+        {
+          title: 'B1',
+          icon: <Image letter='B1' />
+        }, {
+          title: 'B2',
+          icon: <Image letter='B2' />
+        }] },
+    {
+      title: 'C',
+      icon: <Image letter='C' />
+    },
+  ]
+}
+
+const transformParentItem = item => ({
+  ...item,
+  title: 'All of ' + item.title
+})
+
+const StaticExample = () => {
+  return (
+    <NestedSelectModal
+      canSelectParent={true}
+      onSelect={x => x}
+      dismissAction={x => x}
+      isSelected={x => x.title === 'C'}
+      options={options}
+      title="Please select letter"
+      transformParentItem={transformParentItem}
```

Comment:
I still have trouble understanding why we need `transformParentItem` and how to use it. Not sure if it's a naming issues or if it needs more docs either.

Reply:
I think it needed more docs. Tell me if it's clear :) https://github.com/cozy/cozy-ui/pull/1270/commits/72e4fe50db1730fdafd9d1a602af42308de97d0b

Reply:
Yep much better, thanks!